stdlib/2.7/argparse.pyi

```diff
@@ -2,24 +2,24 @@
 #
 # NOTE: This dynamically typed stub was automatically generated by stubgen.
 
-from typing import Any, Sequence, Union
+from typing import Any, Callable, Dict, List, IO, Iterable, Sequence, Union
 
-SUPPRESS = ... # type: Any
-OPTIONAL = ... # type: Any
-ZERO_OR_MORE = ... # type: Any
-ONE_OR_MORE = ... # type: Any
-PARSER = ... # type: Any
-REMAINDER = ... # type: Any
+SUPPRESS = ...  # type: Any
+OPTIONAL = ...  # type: Any
+ZERO_OR_MORE = ...  # type: Any
+ONE_OR_MORE = ...  # type: Any
+PARSER = ...  # type: Any
+REMAINDER = ...  # type: Any
```

Comment:
Please revert all the pure whitespace changes. They're not worth the code churn (there's a piece of advice in the Rails contributions guide about this that I can dig up if you need more explanation).